src/e2e-test/features/dataplex/sink/DataplexSinkErrorScnerios.feature

```diff
@@ -0,0 +1,75 @@
+@Smoke
+@Regression
+Feature: Dataplex-sink - Verify dataplex sink plugin error Scenarios
+
+  @BATCH-TS-DATAPLEX-Error-01
+  Scenario: Verify dataplex sink properties validation errors for mandatory fields with bigquery dataset
+    When Open Datafusion Project to configure pipeline
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Click on the Validate button
+    And Verify mandatory property error for below listed properties:
+    | referenceName|
+    | location     |
+    | lake         |
+    | zone         |
+    | asset        |
+
+
+
+  @BATCH-TS-DATAPLEX-Error-02
+  Scenario: Verify dataplex sink properties for asset details errors with bigquery dataset
+    When Open Datafusion Project to configure pipeline
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Enter input plugin property: "referenceName" with value: "iuhjkhiu"
```

Comment:
Add all data-cy attributes in `pluginDataCyAttributes.properties` file. Add this file in resource folder

Reply:
Use values from `pluginParameters.properties` file